src/ucs/sys/compiler.h

```diff
@@ -95,6 +95,12 @@
 
 /**
  * Size of statically-declared array
+ * May be used for arrays in structure values
+ */
+#define ucs_countof(_array) (sizeof(_array) / sizeof((_array)[0]))
```

Comment:
not needed anymore?

Reply:
removed
ucs_static_array_size is not used too, remove it?
BTW, ucs_static_array_size is failed to compile by Intel compiler (due to static assert inside)

Reply:
yes, let's remove it